gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -73,20 +81,38 @@ func NewVMKeeper(
 	return vmk
 }
 
-func (vm *VMKeeper) Initialize(ms store.MultiStore) {
+func (vm *VMKeeper) Initialize(
+	logger *slog.Logger,
+	ms store.MultiStore,
+	cacheStdlibLoad bool,
+) {
 	if vm.gnoStore != nil {
 		panic("should not happen")
 	}
-	alloc := gno.NewAllocator(maxAllocTx)
 	baseSDKStore := ms.GetStore(vm.baseKey)
 	iavlSDKStore := ms.GetStore(vm.iavlKey)
+
+	if cacheStdlibLoad {
+		vm.gnoStore = CachedStdlibLoad(vm.stdlibsDir, baseSDKStore, iavlSDKStore)
```

Comment:
Shouldn’t you just wrap the existing vm.gnoStore?
Just considering that we’re potentially shadowing something that may give strange results.

Reply:
Nevermind.